tests/integration/test_record.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2016 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+from inspirehep.modules.migrator.tasks import record_upsert, migrate_and_insert_record
+from invenio_indexer.api import RecordIndexer
+from invenio_db import db
+from inspirehep.dojson.hepnames import hepnames, hepnames2marc
+from dojson.contrib.marc21.utils import create_record
+import pytest
```

Comment:
Import blocks should actually be:
```python
from __future__ import absolute_import, division, print_function

import pytest

from dojson.contrib.marc21.utils import create_record
from invenio_db import db
from invenio_indexer.api import RecordIndexer

from inspirehep.dojson.hepnames import hepnames, hepnames2marc
from inspirehep.modules.migrator.tasks import record_upsert, migrate_and_insert_record
```

First `__future__` imports, then stdlib (there isn't any here), then vendored from outside of Invenio, then Invenio, then INSPIRE. A space before each block.